vector/src/main/java/im/vector/app/features/notifications/NotificationEventQueue.kt

```diff
@@ -19,7 +19,14 @@ package im.vector.app.features.notifications
 import timber.log.Timber
 
 class NotificationEventQueue(
-        private val queue: MutableList<NotifiableEvent> = mutableListOf()
+        private val queue: MutableList<NotifiableEvent> = mutableListOf(),
```

Comment:
I'm wondering why `NotificationEventQueue` is not a data class, also why `queue` has default value and `seenEventIds` does not.

Reply:
my original thinking was because the `NotificationEventQueue` is acting as a wrapper around the `List<NotifiableEvent>` and that consumers would use the `rawEvents` for exporting the content instead of comparing equality against the  `NotificationEventQueue` directly

now that you've mentioned it... I can't think any reason not to make it a data class, will change and will also remove the default value to keep all the initialisation in one place :+1:

Reply:
done https://github.com/vector-im/element-android/pull/4506/commits/55fd362b3d7fbc8d3806c5e4bafaab99bb14c703